### ISSUE TITLE
Google Play link is broken + Releases

### ISSUE THREAD

**Issue by dtonon** [aneonex/BitcoinChecker#234]
Google Play link is broken.

Should be nice to have ready to download releases here on GitHub to avoid the Play Store; this works nice with Obtanium too.

**Comment by lybserv2** [comment]
[apk](https://file.io/LCwqwiPADqRv)

would be nice if you can report if it works, dumped it from my main phone to install it on second one, but it force closes when trying to add an exchange. would be good to know before i try to flash another rom ;)

**Comment by dtonon** [comment]
# Warning ⚠️
**Don't download this APK**, the user just joined github and he is not formally related to the app development!

**Comment by lybserv2** [comment]
yeah i expected an answer like this, sorry that i signed up to provide the url. friendly hint: check the md5 if youre paranoid. no one forces you to download it, i just saw that multiple people asked for it & nobody provided the file. cheers.

**Comment by dtonon** [comment]
Thank you for the kindness, but offering an anonymous link is not the proper/educative solution; if an user want to risk with a random apk he can find it online on several (quite shady) websites.
Furthermore, it is not related to the issue that needs to be addressed differently, of course.

>  check the md5

Which one? I cannot find it in the official release.

> if youre paranoid

Paranoid? Are you kidding? This is the basic rule.

Btw the link is broken now, I don't know if you deleted it or the service took it down because of some scanning results.
If you deleted it, let's update your comment.

**Comment by lybserv2** [comment]
I cant upload it on github directly afaik, i provided a dumped apk of the original latest version that was available in play store.
The developer of this app doesnt answer anything for the last like 4 months.

I dont have any md5, i searched for it but i could only find the "old" bitcoin checker versions online.

Just wanted to give people a chance, cause this app is the only one I know with such decent price alerts. 

I can reupload it, didnt take it down - i dont know any viable filehosters anymore. Of course its risky to install apks found online, but yeah - I just wanted to give an "option" for people that are willing to "risk" it, because there is no alternative :)

**Comment by lybserv2** [comment]
https://filebin.net/5lymx4o87alb7irf

**Comment by dtonon** [comment]
>  check the md5 if youre paranoid.

> I dont have any md5

For me this is one big red flag; when someone tries to push a false security when it is actually not possible to achieve it.

By the way, I'm happy if you are in good faith; otherwise, the issue has been flagged and every user will choose what to do :)
